### ISSUE TITLE
GitHub: Comments also look interesting

### ISSUE THREAD

**Issue by Mikaela** [ProgVal/Supybot-plugins#276]
`2015-04-22 11:42:07+0300 <@Dysforia> Mikaela/mikaela.github.io: Mikaela created comment on issue #42: IRC over TLS $issue__url__tiny`

See also:

* https://github.com/ProgVal/Supybot-plugins/issues/274

**Comment by Mikaela** [comment]
Don't forget comments to commits...

```
2015-07-07 10:09:00+0300 <@Dysforia> $repository__owner__name/shell-things (in $ref__branch): $__comment__user__login commented on commit $__commit__message__firstline http://git.io/vq4h4
```

**Comment by nyuszika7h** [comment]
The latter issue has been fixed in f83228e. I will send a PR fixing the other one soon.
